tests/GameVersionTests.cpp

```diff
@@ -267,4 +267,19 @@ TEST(GameVersion_Tests, LessThanOrEqualsFromEquals)
 	const GameVersion versionTwo(1, 2, 3, 4);
 
 	ASSERT_LE(versionTwo, version);
+}
+
+TEST(GameVersion_Tests, VersionEqual)
+{
+	const GameVersion version("1.3.3.0");
+	const GameVersion requiredVersion("1.3.3");
+	ASSERT_EQ(version, requiredVersion);
+	ASSERT_TRUE(version == requiredVersion);
+}
+
+TEST(GameVersion_Tests, VersionNotEqual)
```

Comment:
A name distinguishing this from InequalityCanBeTrueFromFourthPart() would be helpful.

Reply:
Renamed to `GameVersionNotEqualMissingFourthPartIsNotSameAsThirdPart`.